service/gcs/prot/protocol.go

```diff
@@ -932,3 +940,11 @@ type PropertyQuery struct {
 type Properties struct {
 	ProcessList []ProcessDetails `json:",omitempty"`
 }
+
+// LogicalVolume represents a logical volume that is created on top of a set of
+// physical devices.
+type LogicalVolume struct {
+	ID        string
+	MountPath string
+	Devices   []MappedVirtualDiskV2
```

Comment:
I notice `MappedVirtualDiskV2` itself also has a `MountPath`. Do we use that field for something in the LVM case, or should we maybe make a new type that's just the disk identifiers without a mount path, to use here?

Reply:
I'm not sure I care. I can make a new type if you want

Reply:
But I can see how its confusing at best to reuse this type with the precondition that you dont set the MountPoint in the disk itself